config/routes.rb

```diff
@@ -2,6 +2,7 @@
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
-  root 'texts#index'
+  root "texts#index"
   resources :texts
+  resources :movies
```

Comment:
ルーティングは必要なものに制限した方がよいですね。

```suggestion
  resources :texts, only: [:index, :show]
  resources :movies, only: :index
```

Reply:
了解しました！確認ありがとうございます！